Database/EventStore/Internal/Types.hs

```diff
@@ -340,7 +340,7 @@ recordedEventDataAsJson = A.decode . fromStrict . recordedEventData
 --------------------------------------------------------------------------------
 -- | Converts a raw 'Int64' into an 'UTCTime'
 toUTC :: Int64 -> UTCTime
-toUTC = posixSecondsToUTCTime . (/1000) . realToFrac . CTime
+toUTC = posixSecondsToUTCTime . (/1000) . realToFrac . CTime . fromIntegral
```

Comment:
@nrolland Could you indicated briefly the reason of calling `fromIntegral` before `CTime`. A sum up of your PR message. Thanks